pkgs/applications/science/electronics/xyce/default.nix

```
@@ -32,21 +32,21 @@
 assert withMPI -> trilinos.withMPI;
 
 let
-  version = "7.6.0";
+  version = "7.7.0";
 
   # useing fetchurl or fetchFromGitHub doesn't include the manuals
   # due to .gitattributes files
   xyce_src = fetchgit {
     url = "https://github.com/Xyce/Xyce.git";
     rev = "Release-${version}";
-    sha256 = "sha256-HYIzmODMWXBuVRZhcC7LntTysuyXN5A9lb2DeCQQtVw=";
+    sha256 = "1p5kyha8wxv2gk0lfpi5j7avg7f4cmcpj8s5gm03v255lzrhwj8p";
```

Please use NAR hashes

```suggestion
    hash = "sha256-SOMETHING";
```